src/App.css

```diff
@@ -1,13 +1,61 @@
+:root{
+  --primary-color: #4caf50;
+  --error-color: #F44336;
+  --regular-shadow:rgba(149, 157, 165, 0.2) 0px 8px 24px;
+}
+
+*{
+  box-sizing: border-box;
+  margin:0;
+  padding: 0;
+}
+
 body {
   background-color: #fafafa;
 }
 
-.App {
-  width: 100%;
-  max-width: 90vw;
-  margin: 30px auto;
+.todos {
+  width: 80vw;
   display: flex;
-  flex-direction: column;
   align-items: center;
   justify-content: center;
+  flex-direction: column;
+  margin: 30px auto;
```

Comment:
Use `inline-start/end` and `block-start/end` keywords words for `margin/padding` instead of `top/right/bottom/left`.